sdk/core/azure-core-v2/src/main/java/com/azure/v2/core/credentials/AzureTokenRequestContext.java

```diff
@@ -44,36 +43,25 @@
  * @see TokenCredential
  */
 
-public class TokenRequestContext {
-    private final List<String> scopes;
-    private String claims;
+public class AzureTokenRequestContext extends TokenRequestContext {
```

Comment:
We should annotate these classes with the `@Metadata` annotation with fluent specified?

Reply:
Will need to see a sample for Fluent.
For Metadata, do we just mark it @Metadata ?